database/seeds/UserTableSeeder.php

```diff
@@ -54,4 +61,54 @@ public function run()
 
         $customer->save();
     }
+
+    /**
+     * @return array
+     */
+    private function customerData()
+    {
+        return [
+            'title' => 'Mr',
```

Comment:
I appreciate this has been copied across, but can we remove Shaun's details and just fill with nonsense? If you want I can do this since it's our doing :)

Reply:
Yes absolutely :)